partialWaveFit/pwaLikelihood.cc

```diff
@@ -1414,6 +1414,9 @@ pwaLikelihood<complexT>::readWaveList(const vector<waveDescThresType>& waveDescT
 			_waveAmpAdded  [iRefl][iWave] = false;
 			_waveParams.insert(pair<string, pair<unsigned int, unsigned int> >(waveNames[iRefl][iWave], pair<unsigned int, unsigned int>(iRefl, iWave)));
 		}
+
+	printInfo << "set up likelihood function for " << _nmbWaves << " wave" << ((_nmbWaves != 1) ? "s" : "") << " (excluding 'flat' wave)." << endl;
+
```

Comment:
This is nice. Maybe one could add the rank and the number of positive- and negative-reflectvity waves to the printout.